privacyidea/lib/policy.py

```diff
@@ -758,9 +759,11 @@ def ui_get_rights(self, scope, realm, username, client=None):
         elif scope == SCOPE.USER:
             userrealm = realm
             logged_in_user["role"] = ROLE.USER
+            resolver = User(username, userrealm).resolver
         pols = self.get_policies(scope=scope,
                                  adminrealm=adminrealm,
                                  realm=userrealm,
+                                 resolver=resolver,
```

Comment:
Yes.
There is only an *adminrealm*, but no *adminresolver*.
This was not necessary till now. i think this would lead to confusion, when an administrator logs in.
Usually the realms are called like "admins" or "helpdesk". This way the logging in admin knows, what policies or rights he will get.
Otherwise the admin would login as "weber@admin", but could not be sure if he has rights as a superuser or as a servicedesk user.